### PR TITLE
fixs + responsiv

### PR DESCRIPTION
**Pull request by nicolasboeuf**

### CODE REVIEW

File: src/views/InfosDgv.vue [hunk 1]
```diff
@@ -91,18 +92,29 @@ export default {
       return getDataGouvUrl(path)
     },
     prune(string){
+      var lim = Math.round(window.innerWidth/15)
+      console.log(lim)
```
Comment (geoffreyaldebert) [inline src/views/InfosDgv.vue:96]: to remove

**Review by geoffreyaldebert — APPROVED** [review]
can you remove the console.log and comment router with rupturesCaburants before merging !

File: src/views/InfosDgv.vue (hunk 1 above)
Comment (geoffreyaldebert) [inline src/views/InfosDgv.vue:96]:
```suggestion
```

File: src/views/InfosDgv.vue [hunk 2]
```diff
@@ -1,23 +1,25 @@
 <template>
   <div>
     <h2 v-if="display" class="fr-pl-2w fr-py-2w fr-m-0 infosdgv">
```
Comment (nicolaskempf57) [inline src/views/InfosDgv.vue:3]:
```suggestion
    <h2 v-if="display" class="fr-pl-2w fr-py-1v fr-py-m-2w fr-m-0 infosdgv">
```

I think we can remove these paddings on small screens too. What do you think ?

**Pull request merged**